build/packer/teamcity-agent.sh

```diff
@@ -63,6 +63,14 @@ for f in `ls /usr/local/go/bin`; do
     ln -s /usr/local/go/bin/$f /usr/bin
 done
 
+# Install Bazelisk.
```

Comment:
Maybe note here that this should be kept in sync with `build/bazelbuilder/Dockerfile` too?